tests/Pingalot.Core.Tests/PingSessionTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Net.NetworkInformation;
+using System.Net;
+using System.Collections.Generic;
+
+namespace Pingalot.Core.Tests
+{
+	[TestClass]
+	public class PingSessionTests
+	{
+		[TestMethod]
+		public void TestPingStatisticsDivideByZero()
+		{
+			// https://github.com/TurnerSoftware/Pingalot/issues/10
+
+			var pingRequests = new List<PingRequest>();
+
+			var startTime = DateTime.Now;
```

Comment:
I find it is a good practice to use a fixed DateTime when testing - makes tests more predictable. Perhaps use `new DateTime(2022, 5, 19)`.

Reply:
When you set `RequestTime` later on, it can then just be fed the `startTime` variable.

Reply:
I am suggesting we also explicitly set the time? Just to make it super clear what our endTime will be.
`var startTime = new DateTime(2022, 5, 19, 14, 30, 0);`

Reply:
@Turnerj I have add a commit with my code line above that includes time along with fixed date - let me know if any issue.